_includes/markdown/JavaScript.md

```diff
@@ -200,5 +200,7 @@ There are many JavaScript libraries available today. Many of them directly compe
 
 ### Frameworks
 
+[Vue](https://vuejs.org/) - Implementing Vue on a project allows us to take advantage of the statefulness built into something like React, but apply it on a much more lightweight and smaller scale as to not bog down performance by loading in a heavy library. [Learn more about how we use Vue]({{ site.baseurl }}/vue).
```

Comment:
Are we saying Vue is a better tool than React at a smaller scale because of library size?

Reply:
Yeah. For basically one-off things. You don't need Webpack and misc. built tools for it either, so it's a lot easier to integrate within a project.